lib/CL/pocl_mem_management.c

```diff
@@ -1090,3 +1090,24 @@ pocl_find_raw_ptr_with_dev_ptr (cl_context context, const void *dev_ptr)
   POCL_UNLOCK_OBJ (context);
   return item;
 }
+
+/**
```

Comment:
I think this comment belongs to the header since it's a "public function".

Reply:
fixed